plone/app/iterate/tests/test_containers.py

```diff
@@ -51,7 +55,7 @@ def setUp(self):
         )
 
         # add a folder with two documents in it
-        self.portal.invokeFactory("Folder", "docs")
+        self.portal.invokeFactory("LockableFolder", "docs")
```

Comment:
Since `docs` was already a normal folder and is used in other tests, I think it can continue to be a normal folder and the Lockable folder can be created in specific tests.

Reply:
True.  I have fixed this, only now only use `LockableFolder` in one test.